rosa_tutorials/rosa-mobb-sts-rds-database-connect.adoc

```diff
@@ -0,0 +1,399 @@
+   SUBNET_C=$(aws ec2 create-subnet --vpc-id ${VPC_DB} --cidr-block 10.23.3.0/24 --availability-zone ${AWS_REGION}c | jq -r .Subnet.SubnetId)
+----
+
+. Internet gateway:
++
+[source,terminal]
+----
+  $ IGW=$(aws ec2 create-internet-gateway --region ${AWS_REGION} | jq -r .InternetGateway.InternetGatewayId)
+   aws ec2 attach-internet-gateway --vpc-id ${VPC_DB} --internet-gateway-id ${IGW}
+   RT_ID=$(aws ec2 describe-route-tables --filters Name=vpc-id,Values=${VPC_DB} --region ${AWS_REGION}  | jq -r .RouteTables[].RouteTableId)
+   aws ec2 create-route --route-table-id ${RT_ID} --destination-cidr-block 0.0.0.0/0 --gateway-id ${IGW} --region ${AWS_REGION}
+----
+
+. Database subnet group:
++
+[source,terminal]
+----
+   $ aws rds create-db-subnet-group --db-subnet-group-name db-group-${CLUSTER_NAME} \
+      --db-subnet-group-description "DB Subnet group for testing RDS" \
+      --subnet-ids ${SUBNET_A} ${SUBNET_B} ${SUBNET_C} \
+      --region ${AWS_REGION}
+----
+
+== Create RDS Database
+
+. Create database with AWS CLI:
++
+[source,terminal]
+----
+  $ RDS_DB="$(aws rds create-db-instance \
+       --db-instance-identifier psql-${CLUSTER_NAME} \
+       --db-instance-class db.t3.micro \
+       --engine postgres \
+       --master-user-password ${PSQL_PASSWORD} \
+       --allocated-storage 20 \
+       --master-username postgres \
+       --region ${AWS_REGION} \
+       --db-subnet-group-name db-group-${CLUSTER_NAME} \
+       --enable-iam-database-authentication \
+       --publicly-accessible \
+       --region ${AWS_REGION} \
+       | jq -c '.DBInstance | { "DbiResourceId": .DbiResourceId, "VpcSecurityGroups": .VpcSecurityGroups[].VpcSecurityGroupId }')"
+   echo $RDS_DB
+----
+
+. Authorize ROSA cluster to connect to database:
++
+[source,terminal]
+----
+  $ aws ec2 authorize-security-group-ingress \
+    --group-id $(echo $RDS_DB | jq -r .VpcSecurityGroups) \
+    --protocol tcp \
+    --port 5432 \
+    --cidr ${ROSA_IP_OUT}/32 \
+    --region ${AWS_REGION}
+----
+
+== IAM Permissions 
+
+. Build the RDS access policy:
++
+[source,terminal]
+----
+  $ cat <<EOF > $SCRATCH_DIR/rds-policy.json
+   {
+       "Version": "2012-10-17",
+       "Statement": [
+           {
+               "Effect": "Allow",
+               "Action": [
+                   "rds-db:connect"
+               ],
+               "Resource": [
+                   "arn:aws:rds-db:${AWS_REGION}:${AWS_ACCOUNT_ID}:dbuser:$(echo ${RDS_DB} | jq -r .DbiResourceId)/iamuser"
+               ]
+           }
+       ]
+   }
+   EOF
+----
+
+. Create the RDS access policy:
+[NOTE] 
+This creates a named policy for the cluster. You can use a generic policy for multiple clusters.
++
+[source,terminal]
+----
+  $ POLICY=$(aws iam create-policy --policy-name "${CLUSTER_NAME}-rosa-rds-policy" \
+      --policy-document file://$SCRATCH_DIR/rds-policy.json \
+      --query 'Policy.Arn' --output text)
+   echo $POLICY
```

Comment:
Run the two command together will get empty value, unless I run them separatedly